packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -275,7 +274,10 @@ export class OdspDocumentStorageService extends OdspDocumentStorageServiceBase {
                         // Based on the concurrentSnapshotFetch policy:
                         // Either retrieve both the network and cache snapshots concurrently and pick the first to return,
                         // or grab the cache value and then the network value if the cache value returns undefined.
-                        if (this.hostPolicy.concurrentSnapshotFetch && !this.hostPolicy.summarizerClient) {
+                        // For summarizer which could call this a lot due to refreshing of summary parent, always use the cache
```

Comment:
>could call this a lot due to refreshing of summary parent [](http://example.com/codeflow?start=48&length=57)

is it true ? I would expect it not to be called a lot or ever as long as we use the pendngsummaries as we would skip the refresh....

Reply:
I will change the wording. A lot may not be the right word.